src/mcp_atlassian/server.py

```diff
@@ -150,7 +150,7 @@ async def list_resources() -> list[Resource]:
             account_id = ctx.jira.get_current_user_account_id()
 
             # Use JQL to find issues the user is assigned to or reported
-            jql = f"assignee = {account_id} OR reporter = {account_id} ORDER BY updated DESC"
+            jql = f'assignee = "{account_id}" OR reporter = "{account_id}" ORDER BY updated DESC'
```

Comment:
Maybe generate the right escaping? See https://confluence.atlassian.com/jiracoreserver081/search-syntax-for-text-fields-970610224.html 

Or is it not needed because account_id will always be safe?

Reply:
@jeteve Good point! You're right, it's more robust, especially considering potential Server/DC key/name values which might contain special characters. I'll update the PR to handle escaping \ and " correctly before quoting the account_id. Thanks for the catch!